### PR TITLE
Correlations: Hide Correlations Page behind a feature toggle

### PR DESCRIPTION
**Pull request by ifrost**
**What this PR does / why we need it**:

The feature is not ready for the release so we're hiding it behind a feature toggle.

**Which issue(s) this PR fixes**:

Fixes N/A

**Special notes for your reviewer**:

The flag is disabled by default. To enable it add the following to `custom.ini`:

```
[feature_toggles]
correlations = true
```

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -1263,6 +1263,9 @@ default_baselayer_config =
 # Enable or disable loading other base map layers
 enable_custom_baselayers = true
 
+# Correlations page
+correlations = false
```
Comment (joshhunt) [inline conf/defaults.ini:1267]: i don't think this is required fyi. feature toggles are disabled by default 😌

Comment (joshhunt) [reply conf/defaults.ini:1267]: though actually i am a bit confused - this config option is under `geomap`?

Comment (ifrost) [reply conf/defaults.ini]: Removed in 0431ff1ab051d0d7f0f68a407e57c465ee910d4b 👍

**Review by Elfo404 — APPROVED** [review]
🚀

**Pull request merged**